core/services/ardupilot_manager/settings.py

```diff
@@ -0,0 +1,31 @@
+import os
+import pathlib
+
+import appdirs
+
+
+class Settings:
+    app_name = "ardupilot-manager"
+
+    def __init__(self) -> None:
+        # Create folder for ardupilot-manager service if it doesn't exist
+        self.settings_path = appdirs.user_config_dir(Settings.app_name)
+        os.makedirs(self.settings_path, exist_ok=True)
+
+        # Create JSON file for storing mavlink endpoints
+        self.endpoints_path = os.path.join(self.settings_path, "endpoints.json")
+        if not os.path.isfile(self.endpoints_path):
+            with open(self.endpoints_path, "w") as endpoints_file:
+                endpoints_file.write("[]")
```

Comment:
Avoid writing raw strings, in a document that will be json, I believe that is better to write an empty json object

Reply:
Good catch. I will catch the situation where the file is empty.

Reply:
Edit: I will first check if the file is empty, and return an empty array in that case, otherwise it would throw a JSONEncodeError everytime the user has no endpoints.

Reply:
Done